pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-newtest-restapi.md

```diff
@@ -5,146 +5,346 @@ title: IP-Label Newtest Rest API
+| WARNINGSTATUSRED      | Thresholds                                                                                          |                   |             |
+| CRITICALSTATUSRED     | Thresholds                                                                                          |                   |             |
+| EXTRAOPTIONS          | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) | --verbose         |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor the resource using a command like this one (replace the sample values by yours):
 
 ```bash
-/usr/lib/centreon/plugins//centreon_monitoring_iplabel_newtest_restapi.pl \
---plugin=apps::monitoring::iplabel::newtest::restapi::plugin --mode=scenarios \
---hostname='the.newtest.fqdn' --api-username='ro_user' --api-password='strong_psswd' \
---port='443' --proto='https' --http-backend=curl --filter-robot-name='^HELSINKI$' \
---filter-scenario-name='^Sharepoint$'
+/usr/lib/centreon/plugins/centreon_monitoring_iplabel_newtest_restapi.pl \
+    --plugin=apps::monitoring::iplabel::newtest::restapi::plugin \
+    --mode=scenarios \
+    --hostname='the.newtest.fqdn' \
+    --api-username='ro_user' \
+    --api-password='strong_psswd' \
+    --port='443' \
+    --proto='https' \
+    --http-backend=curl \
+    --filter-robot-name='^HELSINKI$' \
+    --filter-scenario-name='^Sharepoint$'
 ```
 
-Expected command output is shown below:
+The expected command output is shown below:
 
-```
+```bash
 OK: Robot 'HELSINKI' scenario 'Sharepoint' green status: 100.00 %, red status: 0.00 %, orange status: 0.00 %, grey status: 0.00 %, execution time: 45000 ms
 Extended status information 	
 checking robot 'HELSINKI'
 scenario 'Sharepoint' green status: 100.00 %, red status: 0.00 %, orange status: 0.00 %, grey status: 0.00 %, execution time: 45000 ms
 ```
 
-The above command checks a specific Newtest Robot at Helsinski (```--filter-robot-name='^HELSINKI$'```) and focus on the Sharepoint scenario (```--filter-scenario-name='^Sharepoint$'```).
+### Troubleshooting
 
-It uses an API account (```--api-username='ro_user' --api-password='strong_psswd'```) and the HTTPS protocol to connect to the IP-Label Newtest API
-(```--proto='https'```).
+Please find the troubleshooting documentation for the API-based plugins in
```

Comment:
```suggestion

#### ```UNKNOWN: 500 Can't connect to the.newtest.fqdn:443```

This error message means that the Centreon Plugin couldn't successfully connect to the IP-Label Newtest API.
Check that no third party device (such as a firewall) is blocking the request.
A proxy connection may also be necessary to connect to the API. This can be done by using the ```--proxyurl``` option in the command.

#### ```UNKNOWN: 501 Protocol scheme 'connect' is not supported |``` 

When using a proxy to connect to the IP-Label Newtest, this error message means that the Centreon Plugin library does not support
the proxy connection protocol.

In order to prevent this issue, use the *curl* HTTP backend by adding the following option to the command: ```--http-backend='curl'```.

#### Generic API troubleshooting

Please find the troubleshooting documentation for the API-based plugins in
```